sensor/DNSProxyPlugin.js

```diff
@@ -245,8 +259,8 @@ class DNSProxyPlugin extends Sensor {
     return intelTool.getDomainIntel(domain);
   }
 
-  async processRequest(qname) {
-    const domain = qnameToDomain(qname);
+  async processRequest(qname, mac) {
+    const domain = qname;
     log.info("dns request is", domain);
     const begin = new Date() / 1;
     const cache = await this.checkCache(domain);
```

Comment:
even if there is a local cache, it should also generate alarm when device accesses domain. The cache is just to skip checking intel from cloud again.